_internal/DeerLister.php

```diff
@@ -256,9 +278,24 @@ public function render(string $directory): string
 
         $title = $path == "" ? "Home" : basename($directory);
         $readme = null;
+        $filesFilter = [];
+        $displayMode = null;
+        $displayBack = true;
+        $displayOthers = true;
+
+        // Check the config to set the display mode and others options set
+        if (isset($this->config["displays"]) && array_key_exists($path, $this->config["displays"]) && array_key_exists($this->config["displays"][$path]["format"], $this->fileDisplays))
```

Comment:
We should really read our config in another way in another merge, to save us from these ugly if statements

Reply:
Yeah we can prob do a class that manage the config file, not sure if it's the scope of this PR tho